lib/ansible/modules/cloud/azure/azure_rm_postgresqldatabase.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_postgresqldatabase
+version_added: "2.5"
+short_description: Manage PostgreSQL Database instance.
+description:
+    - Create, update and delete instance of PostgreSQL Database.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    server_name:
+        description:
+            - The name of the server.
+        required: True
+    name:
+        description:
+            - The name of the database.
+        required: True
+    charset:
+        description:
+            - The charset of the database.
```

Comment:
Is there some sort of source where I can find out valid charset values?

Reply:
i think it just comes directly from postgresql documentation:
https://www.postgresql.org/docs/8.2/static/multibyte.html
so i am not sure whether we should document it here.

Reply:
I would, really helps people who are new to the concept. When adding a url you would enclose it in `U(https://....)`.

Reply:
Well, i am just worried that URL may become outdated at some point, especially  that url includes version (actually here i have included just first URL i found).
Perhaps we should just add "(check PostgreSQL documentation)"?
On the other hand, people who are not familiar with the concept will just use default.
If they need to change charset or collation they would be already aware why they need to change it or what it should be.
Perhaps I will just include some values in sample code.

Reply:
Including values in the sample code would be great, I didn't see the version and in this case maybe just a sentence to tell people where you could find the value, like what you suggested.